WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -172,25 +170,6 @@ - (void)viewWillAppear:(BOOL)animated
     [self reloadData];
 }
 
-- (void)viewWillDisappear:(BOOL)animated
-{
-    [super viewWillDisappear:animated];
-
-    if (self.isStandalone) {
-        if ([Feature enabled:FeatureFlagSyncPublishing]) {
-            return; // No longer needed
-        }
-        if ((self.isBeingDismissed || self.parentViewController.isBeingDismissed) && !self.isStandaloneEditorDismissingAfterSave) {
-            // TODO: Implement it using a ViewModel or a child context to eliminate the risk of accidently saving the changes without uploading them
-            [self.apost.managedObjectContext refreshObject:self.apost mergeChanges:NO];
-        }
-    } else {
-        [self.apost.managedObjectContext performBlock:^{
```

Comment:
⚠️ This is the only changes that's not under the feature flag. There is no need to save as the editor manages saving.